client/src/components/Footnote.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import styled from "styled-components";
+
+const StyledFooter = styled.div`
+  position: absolute;
+  width: 376px;
```

Comment:
Let's use `rem`  values instead of `px`. For example, right now we have `1rem = 10px`. So for line 8, it would be `37.6rem` because 376px/10px = 1